packages/ckeditor5-link/src/linkui.ts

```diff
@@ -197,6 +197,7 @@ export default class LinkUI extends Plugin {
 			const { value } = formView.urlInputView.fieldView.element!;
 			const parsedUrl = addLinkProtocolIfApplicable( value, defaultProtocol );
 			editor.execute( 'link', parsedUrl, formView.getDecoratorSwitchesState() );
+			formView.urlInputView.fieldView.reset();
```

Comment:
It looks like we should also add this to the `cancel` and `Esc` listeners. Otherwise, if a user inserts a value in the input field and then chooses Cancel/Esc, the next time the form is opened, the `Save' button will be active.